runtime/doc/repeat.txt

```diff
@@ -139,17 +139,78 @@ q			Stops recording.  (Implementation note: The 'q' that
 			result of evaluating the expression is executed as an
 			Ex command.
 			Mappings are not recognized in these commands.
-			{Vi: only in some versions} Future: Will execute the
-			register for each line in the address range.
+
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+
+			TODO(Pyrohh): what does the below line mean? I don't
+			know what is meant by "future", as I'm not sure if
+			what is mentioned below is already implemented.
```

Comment:
No, it isn't implemented yet but it's in the todo list:

> The "`:@r`" commands should take a range and execute the register for each line in the range.

Personally I would leave it in and deal with it some other time.